main/include/OpenThreadConfig.h

```diff
@@ -30,9 +30,17 @@
 // openweave/src/adaptations/device-layer/nRF5/Logging.cpp).
 #define OPENTHREAD_CONFIG_LOG_OUTPUT OPENTHREAD_CONFIG_LOG_OUTPUT_APP
 
-// Turn on a moderate level of logging in OpenThread
+// Turn on a moderate level of logging in OpenThread.
+// Set to OT_LOG_LEVEL_INFO or OT_LOG_LEVEL_DEBG for more verbose output if experiencing
+// issues with Thread.
 #define OPENTHREAD_CONFIG_LOG_LEVEL OT_LOG_LEVEL_NOTE
 
+// When operating in a less than ideal RF environment, having a more forgiving configuration
+// of OpenThread makes thread a great deal more reliable.
+#define OPENTHREAD_CONFIG_MAC_MAX_FRAME_RETRIES_DIRECT 15 // default is 3
```

Comment:
Why do we think these are the right settings?  What are the implications on battery lifetime?  How do these compare with the settings used for shipping devices?

Reply:
These were suggested by RS after he went through some lengthy troubleshooting (I believe). I also believe some of these values mirror what's been used by shipping devices. No clue for impact on battery lifetime.